<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Antispam::SettingsWidget</class>
 <widget class="QWidget" name="Antispam::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="enabledBox">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Question:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="questionEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Answers (semicolon as a separator):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="answerEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Message on correct answer:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="successEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
